kernel/Cargo.toml

```
@@ -56,7 +54,7 @@ enable-gdb = ["dep:gdbstub", "dep:gdbstub_arch"]
 vtpm = ["dep:libtcgtpm"]
 nosmep = []
 nosmap = []
-verus_all = ["builtin", "verify_proof/verus", "verify_external/verus"]
+verus_all = ["verify_proof/verus", "verify_external/verus"]
 verus = ["verus_all", "verify_proof/noverify", "verify_external/noverify"]
```

Maybe we should move the set of features of dependencies in the new crate and here just enabled the new features you added in the wrapper crate.

---

Nope, `kernel` is still using those, so they should stay here.

---

```
builtin = { workspace = true, optional = true}
```

builtin crate should be optionally added since verus macro generated spec/proof code will use builtin.

```suggestion
verus_all = ["builtin", "verify_proof/verus", "verify_external/verus", "verus/enable"]
verus = ["verus_all", "verify_proof/noverify", "verify_external/noverify"]
```